src/d/component.spec.ts

```diff
@@ -1,25 +1,31 @@
 import { Component, User } from './component';
 
-const mockData: User = { id: 'foo', name: 'bar', age: 40 };
-
 describe('Component', () => {
     let component: Component;
     beforeEach(() => {
         component = new Component();
-        component.setUser(mockData);
     });
 
+    function mockUser(): User {
+        return <User>{ id: 'foo', name: 'bar', age: 40 };
+    }
```

Comment:
i prefer using arrow functions but this is just preference
```suggestion
    const mockUser = (): User => <User>{ ... }
```

Reply:
could also pass in the age and this allows us to set the age per test
```suggestion
    function mockUser(age: number): User {
        return <User>{ id: 'foo', name: 'bar', age };
    }
```

Reply:
though no changes actually required, just flagging other approaches